assets/css/edd-admin.css

```diff
@@ -57,14 +57,14 @@ a.edd-delete:hover {
 	z-index: 1;
 }
 
-.edd-from-to-wrapper input[name="start"],
+.edd-from-to-wrapper input[name*="export-start"],
```

Comment:
Nice suggestion. I had done it so that I wouldn't accidentally trigger any changes, but I did some digging and was even able to shorten this CSS block by a selector.